hw/ip/pwm/data/ot_regtool

```diff
@@ -0,0 +1 @@
+/wdc/proj/cto/top/work/${USER}/opentitan/util/regtool.py -r -t ../rtl pwm.hjson
```

Comment:
You can add pwm to IPS in `hw/Makefile` then you can generate pwm by running `make -C hw pwm_reg`. I think this file is not needed.

Reply:
This is ../ot_regtool/ file we use in WD ENV that I should not have included. I will delete it